metron-platform/metron-writer/src/main/java/org/apache/metron/writer/WriterToBulkWriter.java

```diff
@@ -44,11 +45,15 @@ public void init(Map stormConf, WriterConfiguration config) throws Exception {
   }
 
   @Override
-  public void write(String sensorType, WriterConfiguration configurations, Iterable<Tuple> tuples, List<MESSAGE_T> messages) throws Exception {
+  public BulkWriterResponse write(String sensorType, WriterConfiguration configurations, Iterable<Tuple> tuples, List<MESSAGE_T> messages) throws Exception {
     if(messages.size() > 1) {
       throw new IllegalStateException("WriterToBulkWriter expects a batch of exactly 1");
     }
     messageWriter.write(sensorType, configurations, Iterables.getFirst(tuples, null), Iterables.getFirst(messages, null));
+
+    BulkWriterResponse response = new BulkWriterResponse();
```

Comment:
What if there's an error writing?  Do we still want to add the tuples to the response as successes?